drivers/video/video.c

```diff
@@ -667,6 +669,7 @@ static int start_capture(enum v4l2_buf_type type,
   convert_to_imgsensorfmt(&fmt[VIDEO_FMT_SUB], &sf[IMGSENSOR_FMT_SUB]);
   convert_to_imgsensorinterval(interval, &si);
 
+  *seqnum = 0;
```

Comment:
look like it's better to zero at caller site, sorry please ignore my previous comment.

Reply:
sure